src/storage/sftp/storage.c

```diff
@@ -1098,7 +1127,7 @@ storageSftpNew(
             THROW_FMT(ServiceError, "timeout during libssh2 handshake [%d]", rc);
 
         if (rc != 0)
-            THROW_FMT(ServiceError, "libssh2 handshake failed [%d]", rc);
+            THROW_FMT(ServiceError, "libssh2 handshake failed [%d]; man libssh2_session_handshake to map error", rc);
```

Comment:
Shouldn't the ; be a :? Also, not sure what `man libssh2_session_handshake to map error` means.

Reply:
I updated the handshake error handling to get  the error message from libssh2, replacing the poor attempt to refer to the man page to map the error code